src/migrations/hotbar-store/5.0.0-alpha.0.ts

```diff
@@ -21,33 +21,18 @@
 
 // Cleans up a store that had the state related data stored
 import type { Hotbar } from "../../common/hotbar-store";
-import { ClusterStore } from "../../common/cluster-store";
 import * as uuid from "uuid";
 import type { MigrationDeclaration } from "../helpers";
 
 export default {
   version: "5.0.0-alpha.0",
   run(store) {
-    const hotbars: Hotbar[] = [];
+    const hotbar: Hotbar = {
+      id: uuid.v4(),
+      name: "default",
+      items: [...Array.from(Array(12).fill(null))],
+    };
 
-    ClusterStore.getInstance().clustersList.forEach((cluster: any) => {
-      const name = cluster.workspace;
-
-      if (!name) return;
-
-      let hotbar = hotbars.find((h) => h.name === name);
-
-      if (!hotbar) {
-        hotbar = { id: uuid.v4(), name, items: [] };
-        hotbars.push(hotbar);
-      }
-
-      hotbar.items.push({
-        entity: { uid: cluster.id },
-        params: {}
-      });
-    });
-
-    store.set("hotbars", hotbars);
```

Comment:
This migration just doesn't make any sense... so I replaced it with one that makes sure that there is an initial hotbar.